samples/plugins/postgres_jdbc/manifest.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
 <connector-plugin class='postgres_jdbc' superclass='jdbc' plugin-version='0.0.0' name='PostgreSQL JDBC' version='18.1'>
+  <vendor-information>
+      <company name="Company Name"/>
+      <support-link url = "support-link-url"/>
```

Comment:
We should use http://example.com for both connectors so that they can click on the link and not get an error.